packages/ui/src/machines/authenticator/defaultAuthServices.ts

```diff
@@ -0,0 +1,36 @@
+import { Auth } from 'aws-amplify';
+export function signUp(username, password, attributes) {
+  return Auth.signUp({
+    username,
+    password,
+    attributes,
+    autoSignIn: {
+      enabled: true,
+    },
+  });
+}
+
+export function signIn(username, password) {
+  return Auth.signIn(username, password);
+}
+
+export function confirmSignIn(user, code, mfaType) {
+  return Auth.confirmSignIn(
+    user,
+    code,
+    // cast due to restrictive typing of Auth.confirmSignIn
+    mfaType as 'SMS_MFA' | 'SOFTWARE_TOKEN_MFA'
+  );
+}
```

Comment:
Wonder we should type the parameters instead so that we don't have to these casting.